types/phaser.d.ts

```diff
@@ -5601,45 +5601,51 @@ declare namespace Phaser {
             getPoints(divisions?: integer): Phaser.Math.Vector2[];
 
             /**
-             * [description]
+             * Returns a randomly chosen point anywhere on the path. This follows the same rules as `getPoint` in that it may return a point on any Curve inside this path.
```

Comment:
You cannot edit the defs file directly, as all of your changes will be automatically overwritten the next time the defs file is generated. Please edit the jsdocs instead, then those changes will filter down to the defs.

Reply:
I regenerated this by `tsdev` command

Reply:
That makes sense - please remove the defs file from your PR so I can merge it :)